dipy/segment/tests/test_clustering.py

```diff
@@ -445,20 +479,36 @@ def test_cluster_map_get_small_and_large_clusters():
     cluster_map = ClusterMap()
 
     # Randomly generate small clusters
-    indices = [rng.randint(0, 10, size=i) for i in range(1, nb_clusters+1)]
+    indices = [rng.randint(0, 10, size=i) for i in range(1, nb_clusters + 1)]
     small_clusters = [Cluster(indices=indices[i]) for i in range(nb_clusters)]
     cluster_map.add_cluster(*small_clusters)
 
     # Randomly generate small clusters
-    indices = [rng.randint(0, 10, size=i) for i in range(nb_clusters+1, 2*nb_clusters+1)]
+    indices = [
+        rng.randint(
+            0,
+            10,
+            size=i) for i in range(
+            nb_clusters +
+            1,
+            2 *
+            nb_clusters +
+            1)]
```

Comment:
This line will be easier to read if you break it at the `for`:

    indices = [rng.randint(0, 10, size=i) 
                    for i in range(nb_clusters+1, 2*nb_clusters+1)]